spec/lib/account_spec.rb

```diff
@@ -0,0 +1,73 @@
+require 'spec_helper'
+
+describe BankAccounts::Account do
+
+  describe "class methods" do
+    it "responds to 'new'" do
+      expect(BankAccounts::Account).to respond_to :new
+    end
+
+    let(:account) do
```

Comment:
Heh, I had it that way and another TA suggested that I use do/end because the lines were so long. :) I gather that either way is acceptable...